docs/7-infrastructure-configuration-management/7.1.3-terraform-modules.md

```diff
@@ -117,12 +138,12 @@ module "vpc" {
 
 module "ec2_instance" {
   source  = "terraform-aws-modules/ec2-instance/aws"
-  version = "4.3.0"
+  version = "5.6.1"
 
   count = 1
   name  = "my-ec2-cluster-${count.index}"
 
-  ami                    = "ami-0c5204531f799e0c6"
+  ami                    = "ami-0fe630eb857a6ec83"
```

Comment:
What AMI did you change to? Would be nice to add a comment after the ami to let future maintainers know. Something like `# Amazon Linux 2`

Reply:
Got it, just updated in the most recent commit # Red Hat Enterprise Linux 9